services/utils/__init__.py

```diff
@@ -101,24 +102,68 @@ def __init__(self,
                  pool_min: int = 1,
                  pool_max: int = 10,
                  timeout: int = 60):
-        table = str.maketrans({"'": "\'", "`": r"\`"})
```

Comment:
Removing this for a more general-purpose translation of user, password, host, and port via Postgres connection string and DSN; The DSN doesn't require escaping when we give to `aiopg` ; For [goose](https://github.com/pressly/goose#usage) we provide PG connection string which is supported after [Postgres version 9.2](https://www.postgresql.org/docs/9.2/libpq-connect.html#LIBPQ-CONNSTRING).